frontend/src/utils/game/cameraSetting.js

```diff
@@ -0,0 +1,34 @@
+import * as THREE from 'three';
+
+function cameraSetting(mode, side) {
+  const fov = mode === 'local' ? 60 : 45;
```

Comment:
```suggestion
  const fov = mode === 'local' ? 65 : 45;
```
로컬일 때 좀 꽉 차보여서 fov값을 65정도로 주는건 어떨까요?